### PR TITLE
upadte 2d20 roll and damage roll 

### PR DESCRIPTION
**Pull request by bb46003**
change for 2d20 roll :
 - add automatization for buing d20 dice including PC and GM, allow to buy dice for GM AP
- add button for assistance, always will roll 1d20
- reduce ammo for GM (option i settings)
- reduce 10 amo for gatling guns

change for damage roll :
- for guns fire rate can be add at begining of rolling, read weapon property and chcek if we have enough amo to shoot more bullet
- for mele/unarmed can spend max 3 AP for PC and GM
- reduce AP if avaliable for both PC and GM
- for PC buy from GM, with notification, add to GM AP automaticaly 
- add button from chat read weapon fire rate and chcek avaliable amo

### CODE REVIEW

**Comment by Muttley** [comment]
I'll try and take a look at this after the next release.  I think it looks like it might need a bit of beautification before it can go into a release, judging by the screenshots you've shared.

**Comment by bb46003** [comment]
I was not spending much time on css just to look reasonable fine

File: i18n/pl.yaml
Comment (Muttley) [inline i18n/pl.yaml:1]: Please don't check in changes to the non `en.yaml` translation files, any changes will be overwritten with the next sync from Crowdin anyway.

File: i18n/en.yaml [hunk 1]
```diff
@@ -491,10 +491,16 @@ FALLOUT.TOOLTIPS.WeaponQuality.unreliable: When you make an attack with an Unrel
 FALLOUT.TOOLTIPS.WeaponQuality.unstable_radiation: This was never designed to be used as a handheld weapon. When attacking with a weapon, each complication you inflicts 2DC Radiation damage from radioactive feedback. A weapon may not have both Reliable and Unstable Radiation.
 FALLOUT.UI.Add: Add
 FALLOUT.UI.Addictions: Addictions
+FALLOUT.UI.Additionalamo: Additional damage dice from firerate
+FALLOUT.UI.Additionalmeledmg: Additional damage dice for party AP
+FALLOUT.UI.AdditionalmeledmgOverseer: Additional damage dice for overseer AP
```
Comment (Muttley) [inline i18n/en.yaml:496]: As mentioned in your other pull request, please try and keep keys readable. Going forward I'm trying to standardise the i18n key style to snake_case, avoiding unecessary abbreviations where possible

File: system/src/roller/Dialog2d20.mjs
Comment (Muttley) [inline system/src/roller/Dialog2d20.mjs:1]: There are several instances in this file where function/method names are hard to read being all lower-case with multiple words merged together.

Please make sure all function/methods names in `camelCaseFormat`.

Also, any new html being rendered should be done via relevant template files, not embedded in code please.  I'm gradually stripping out existing instances of that.

File: system/src/roller/DialogD6.mjs
Comment (Muttley) [inline system/src/roller/DialogD6.mjs:1]: There are similar problems in this file as mentioned in the comment for `Dialog2d20.mjs`

File: system/src/system/FalloutChat.mjs [hunk 2]
```diff
@@ -132,7 +132,7 @@ export default class FalloutChat {
 				const falloutRoll = message.flags.falloutroll;
 				const weapon = message.flags.weapon;
 
-				fallout.DialogD6.createDialog({
+				fallout.DialogD6.addcreateDialog({
```
Comment (Muttley) [inline system/src/system/FalloutChat.mjs:135]: Method names should be `camelCaseStyle`

File: system/system.json
Comment (Muttley) [inline system/system.json:1]: This change breaks the `manifest` config settings.  Please don't make changes to `system.json`

**Review by Muttley — CHANGES_REQUESTED** [review]
I'm afraid you'll need to resolve any problems commented on in this code before I can safely merge it.

**Comment by bb46003** [comment]
I do all mentioned request, as well fixing few issue that I recognie like macros for rollind d6 or 2d20

**Comment by Muttley** [comment]
Having had a play around with this, I really don't like the workflow changes this makes to the dice rolling, as it feels quite clunky and unintuitive.

I'm afraid I'm going to have to reject this one as it would be more work to resolve the issues with this implementation than it would be to write the new dice roller I'd already envisioned replacing the existing one.

**Pull request closed without merging**